schemas/blocks/interstitial.json

```diff
@@ -23,15 +23,13 @@
         "enum": ["Interstitial"]
       },
       "questions": {
-        "type": "array",
-        "minItems": 1,
-        "items": {
-          "oneOf": [
-            {
-              "$ref": "../questions/content.json#/question"
-            }
-          ]
-        }
+        "$ref": "../questions/definitions.json#/content_questions"
+      },
+      "question": {
+        "$ref": "../questions/definitions.json#/question"
```

Comment:
Need a `content_question`?

Reply:
I didn't want to remove backwards compatibility for question on interstitial at this point.